client/component/src/selections/utils.tsx

```diff
@@ -592,7 +592,7 @@ function useSelections(initSelections?: SelectionBase[]) {
  * Set fixed and asDashed properties of selection to false.
  * @param {SelectionBase} s - The selection to modify.
  */
-function enableSelection(s: SelectionBase) {
+function disableSelection(s: SelectionBase) {
```

Comment:
If the newer disableSelection definition is preferred (`s.fixed = true, s.asDashed = true`) then all the instances where they are used can be switched instead.